packages/xstate-store/src/fromStore.ts

```diff
@@ -97,17 +93,11 @@ export function fromStore<
     | ({
         context: ((input: TInput) => TContext) | TContext;
         on: {
-          [K in keyof TEventPayloadMap & string]:
-            | StoreAssigner<
-                NoInfer<TContext>,
-                { type: K } & TEventPayloadMap[K],
-                Cast<TTypes['emitted'], EventObject>
-              >
-            | StorePropertyAssigner<
-                NoInfer<TContext>,
-                { type: K } & TEventPayloadMap[K],
-                Cast<TTypes['emitted'], EventObject>
-              >;
+          [K in keyof TEventPayloadMap & string]: StoreAssigner<
+            NoInfer<TContext>,
+            { type: K } & TEventPayloadMap[K],
+            Cast<TTypes['emitted'], EventObject>
+          >;
         };
       } & { types?: TTypes }),
```

Comment:
should `types` property still be supported here?

Reply:
and similarly, should `fromStore` still support `initialContextOrObj` parameter?

Reply:
Fixed